### PR TITLE
Add option to pretty print JSON responses in airnode-rrp-log

### PR DESCRIPTION
**Pull request by dcroote**
I thought it would be useful to add the ability to pretty print the JSON responses. I've tested it works, but I'm not sure the way I've achieved it is exactly idiomatic rust.

### CODE REVIEW

File: airnode-rrp-log/src/main.rs [hunk 1]
```diff
@@ -26,15 +26,19 @@ impl State {
 }
 
 impl reader::EventHandler for State {
-    fn on(&mut self, l: web3::types::Log) -> () {
+    fn on(&mut self, l: web3::types::Log, pretty_print: bool) -> () {
```
Comment (EnormousCloud) [inline airnode-rrp-log/src/main.rs:29]: Could you please move `pretty_print` out of the handler into `State` properties which are set in its constructor? Everything else LGFM

**Review by EnormousCloud — CHANGES_REQUESTED** [review]

File: airnode-rrp-log/src/main.rs [hunk 2]
```diff
@@ -14,13 +15,15 @@ pub struct State {
...
+    pub pretty_print: bool,
 }
 
 impl State {
-    pub fn new() -> Self {
+    pub fn new(args: &Args) -> Self {
         Self {
             unknown: BTreeMap::new(),
             filtration: LogFiltration::default(),
+            pretty_print: args.pretty_print,
```
Comment (dcroote) [inline airnode-rrp-log/src/main.rs:26]: It looks like `LogFiltration` isn't getting any of the arguments? I've tested by specifying `--by-request-id 0xd2de59b0b33b15b605451480a56977be14137c02aedbd39b258a17c86e0dde3d` and the output is the same as not specifying that filter.

Comment (EnormousCloud) [reply airnode-rrp-log/src/main.rs:26]: its seems you are right, thanks for noticing
It will be fixed

Comment (dcroote) [reply airnode-rrp-log/src/main.rs:26]: Ok great!

**Pull request merged**